.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+## Description
+<!--
+PR to add a new Kotlion sample _"SAMPLE_NAME"_ in `SAMPLE_CATEGORY` category.
+-->
+## Links and Data
+<!--
+Sample Epic: `runtime/common-samples/issues/ISSUE_NUMBER`
+-->
+## What To Review
+<!--
+-  Review the code to make sure it is easy to follow like other samples on Android
+- `README.md` and `README.metadata.json` files
+-->
+
+## How to Test
+<!--
+Run the sample on the sample viewer or the repo.
+-->
+
+<!--
+## To Discuss
+-->
+
+<!--
+## Screenshots
```

Comment:
These two headings are currently commented out I think

Reply:
They are intentionally commented out, as they are optional sections to be used in a PR. The ones that are not commented out are meant to be sections that are required to be filled in for the PR.

Reply:
Added an `OPTIONAL` tag to the comment to clarify this.